bucket/visualvm.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "2.0.2",
+    "description": "Java troubleshooting tool",
+    "homepage": "https://visualvm.github.io/index.html",
+    "license": "GPL-2.0-only",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://github.com/oracle/visualvm/releases/download/2.0.2/visualvm_202.zip",
+    "hash": "e189e847cf299d0e5336690428b95ad1b9f3b88d4502bfc67e316502b56af56f",
+    "extract_dir": "visualvm_202",
+    "pre_install": [
+        "if (Test-Path \"$persist_dir\\etc\\visualvm.conf\") {",
+        "    Remove-Item -Recurse \"$dir\\etc\"",
+        "    $conf = Get-Content \"$persist_dir\\etc\\visualvm.conf\"",
+        "    $appdir = \"$((Split-Path $dir) -replace '\\\\', '\\\\')\\\\[\\d.]+\"",
+        "    $conf = $conf -replace \"^(visualvm_default_userdir=)`\"$appdir\\\\user`\"$\", \"`$1\"\"$dir\\user`\"\"",
+        "    $conf = $conf -replace \"^(visualvm_default_cachedir=)`\"$appdir\\\\cache`\"$\", \"`$1\"\"$dir\\cache`\"\"",
+        "    Set-Content \"$persist_dir\\etc\\visualvm.conf\" -Value $conf -Encoding Ascii",
+        "} else {",
+        "    $conf = $conf -replace '^(visualvm_default_userdir=).*$', \"`$1\"\"$dir\\user`\"\"",
+        "    $conf = $conf -replace '^(visualvm_default_cachedir=).*$', \"`$1\"\"$dir\\cache`\"\"",
+        "    if ($env:JAVA_HOME -and (Test-Path $env:JAVA_HOME)) { $conf = $conf -replace '^#(visualvm_jdkhome=).*$', \"`$1\"\"$env:JAVA_HOME`\"\" }",
+        "    Set-Content \"$dir\\etc\\visualvm.conf\" -Value $conf -Encoding Ascii",
+        "}"
```

Comment:
What the hell?

What are you trying to do?

Reply:
VisualVM fails to start for me when I attempt to use the `current` junction.